apps/extension/src/ui/apps/popup/components/TotalFiatBalance.tsx

```diff
@@ -44,30 +42,24 @@ export const TotalFiatBalance = ({ className, mouseOver }: Props) => {
   return (
     <div className={classNames("flex flex-col items-start justify-center gap-4", className)}>
       <div className="text-body-secondary mt-2 flex gap-2 text-sm">
-        {hasAccounts ? (
-          <>
-            <span>{t("Total Portfolio")}</span>
-            <button
-              className={classNames(
-                "hover:text-body focus:text-body pointer-events-auto opacity-0 transition-opacity",
-                (hideBalances || mouseOver) && "opacity-100"
-              )}
-              onClick={toggleHideBalance}
-            >
-              {hideBalances ? <EyeIcon /> : <EyeOffIcon />}
-            </button>
-          </>
-        ) : (
-          <span>{t("No accounts")}</span>
-        )}
+        <span>{t("Total Portfolio")}</span>
+        <button
+          className={classNames(
+            "hover:text-body focus:text-body pointer-events-auto opacity-0 transition-opacity",
+            (hideBalances || mouseOver) && "opacity-100"
+          )}
+          onClick={toggleHideBalance}
+        >
+          {hideBalances ? <EyeIcon /> : <EyeOffIcon />}
+        </button>
       </div>
       <Fiat
         className="font-surtExpanded text-lg"
         amount={balances.sum.fiat("usd").total}
         currency="usd"
         isBalance
       />
-      {hasAccounts && <TopActions />}
+      <TopActions />
```

Comment:
might want to hide these if no accounts

Reply:
This entire component will no longer be shown if no accounts, so it isn't needed here

Reply:
it did appear for me when i tested, with no account